components/alertbanner/metadata/alertbanner.yml

```diff
@@ -0,0 +1,116 @@
+name: Alert Banner
+description: Alert banners show pressing and high-signal messages, such as system alerts. They're meant to be noticed and prompt users to take action.
+sections:
+  - name: Custom Properties API
+    description: |
+      This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a href="https://github.com/adobe/spectrum-css/tree/main/components/alertbanner/metadata/mods.md">here</a>.
```

Comment:
You can remove this section - we'll be launching the new 11ty soon and I've gone through and removed these from the docs.